ayon_api/server_api.py

```diff
@@ -568,6 +572,29 @@ def set_default_settings_variant(self, variant):
         set_default_settings_variant
     )
 
+    def get_sender(self):
+        """Sender used to send requests.
+
+        Returns:
+            Union[str, None]: Sender name or None.
+        """
+
+        return self._sender
+
+    def set_sender(self, sender):
+        """Change sender used for requests.
+
+        Args:
+            sender (Union[str, None]): Sender name or None.
+        """
+
```

Comment:
We are not handling if sender is `None`,  so this would make all calls to be sender `null` , and I cant seem to find it handled in `ayon-backend` ?

Reply:
It is in `_update_session_headers`.